.github/workflows/ci.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@aaa42aa0628b4ae2578232a66b541047968fac86 # v6.1.0
         with:
-          version: v1.57.2
+          version: v1.63.4
```

Comment:
Is this upgrade related to adding linter rules to check for regression to direct use of `armon/go-metrics`?

Reply:
No it was a random attempt at getting the linter to run in CI. Before the linter would start and it would time out waiting for it to do anything.

I just added the extra linter rules as this library might see a bit more active development.